app/pages/mccloud/AddAnotherPensionSchemePage.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.mccloud
+
+import models.ChargeType
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+
+case class AddAnotherPensionSchemePage(chargeType: ChargeType, index: Int, schemeIndex: Int) extends QuestionPage[Boolean] {
+  override def path: JsPath =
+    JsPath \ ChargeType.chargeBaseNode(chargeType) \ "members" \ index \ "mccloudRemedy" \ "schemes" \ schemeIndex \ AddAnotherPensionSchemePage.toString
```

Comment:
Consider using the new SchemesQuery object to save duplicating code here. Same on EnterPstrPage. Possible rather than creating a case class instance only to create a path you could change SchemesQuery to be a singleton object with a method.